public/static/docs/command-reference/status.md

```diff
@@ -32,12 +32,12 @@ The two modes, _local_ and _cloud_ are triggered by using the `--cloud` or
 | remote | `--cloud`  | Comparisons are made between the cache, and the default remote, defined with `dvc remote --default` command.                |
 
 DVC determines data and code files to compare by analyzing all
-[DVC-files](/doc/user-guide/dvc-file-format) in the <abbr>project</abbr>
-(`--all-branches` and `--all-tags` in the `cloud` mode compare multiple
-workspace versions). The comparison can be limited to specific DVC-files by
-listing them as `targets`. Changes are reported only against the given
-`targets`. When combined with the `--with-deps` option, a search is made for
-changes in other stages that affect the target.
+[DVC-files](/doc/user-guide/dvc-file-format) in the <abbr>repository</abbr>
+(`--all-branches` and `--all-tags` in `--cloud` mode compare to multiple
```

Comment:
I would remove `to`

Reply:
Agree. I also removed "in `--cloud` mode" – that has nothing to do with Git. In fact I rephrased this whole paragraph, and split it in two. Please see b50d6a9a